README.md

```diff
@@ -28,6 +28,7 @@ Session data is _not_ saved in the cookie itself, just the session ID.
 #### Options
 
   - `name` - cookie name (formerly known as `key`). (default: `'connect.sid'`)
+  - `headerName` - optional HTTP header name to pass session ID, e.g. `X-Session-Token`. (default: `undefined`)
```

Comment:
in index.js
var signed = 's:' + signature.sign(val, secret);

secret is an Array, instead of a string, so your code has broken.